libs/util/string.hpp

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
don't like naming. When people write writing #include string.hpp looks like including system library

Reply:
they'll include `util/string.hpp` in iroha sources or smth like `iroha/util/string.hpp` outsourced
imo that's fine

Reply:
Yes, folder `util` works as namespace.